db/migrate/20240530002641_change_credit_card_number_to_bigint_in_transactions.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeCreditCardNumberToBigintInTransactions < ActiveRecord::Migration[7.1]
+  def change
+    change_column :transactions, :credit_card_number, :bigint
```

Comment:
right! b/c as an integer the cc number is over the size limit for bytes. Changing to bigint allows it to hold larger numbers at a smaller size due to using scientific notation (I'm guessing)